### PR TITLE
add tests for except operator

### PR DESCRIPTION
**Pull request by olvidalo**
as requested in #1348

### CODE REVIEW

File: test/src/xquery/exceptOperator.xql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+declare
+ %test:tearDown 
+ function excepttest:cleanup() { 
+ xmldb:remove($excepttest:COLLECTION_NAME)
+ }; 
+
+
+declare
+%test:assertEquals("Act 1")
```
Comment (dizzzz) [inline test/src/xquery/exceptOperator.xql]: I am not sure if the tested result is OK

File: test/src/xquery/exceptOperator.xql [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+function excepttest:except-wildcard() {
+ ($excepttest:COLLECTION/* except $excepttest:SUBCOL2/*)//tei:head/text()
+};
+
+declare
+%test:assertEquals("Act 1")
```
Comment (dizzzz) [inline test/src/xquery/exceptOperator.xql]: I am not sure if the tested result is OK

**Comment by olvidalo** [comment]
> I think the test is not so strong. Having a look at https://www.w3.org/TR/xpath-30/#combining_seq (with the A,B,C example) we could be able to have a better test.

This test was not meant to be an exhaustive test for the except operator. I made it solely for demonstrating an issue I had that I reported in #1348, which has since be resolved. @duncdrum thought it would be beneficial to have it in the test suite to catch possible regressions, so I made this PR. I really don't mind what you do with it :)

**Comment by adamretter** [comment]
Thanks for your contribution @olvidalo 

Requires https://github.com/eXist-db/exist/pull/2140 first

**Comment by duncdrum** [comment]
@dizzzz with 2140 in the green I m inclined to merge this. What's in this PR is better than nothing. Improvements are of course always welcome, and can be added later.

**Comment by adamretter** [comment]
@olvidalo #2140 is now merged. Could please rebase this on the develop HEAD? We can then look to get this PR merged.

**Comment by olvidalo** [comment]
done

**Pull request merged**